### DISCUSSION TITLE
Night mode

### DISCUSSION THREAD

**Discussion by rochat2** [Sketchware-Pro/Sketchware-Pro#837] [Ideas]
Can you add night mode in the sketchware app?  since the eyes get tired with light colors

**Comment by DevVigilante** [comment]
Use new theme(But you can't import block and create new projects)
ScreenShot
![Screenshot_2023-04-09-09-46-30-065_com sketchware remod](https://user-images.githubusercontent.com/96680248/231133515-546c285f-a13e-4ce3-8d0b-7d3b9aadfacd.jpg)
